client/guidestours/steps.js

```diff
@@ -38,10 +38,10 @@ class GuidesFirstStep extends Component {
 
 		const { text, onNext, onQuit } = this.props;
 		return (
-			<Card className="guidestours__step" style={ stepCoords } >
+			<Card className="guidestours__step guidestours__step-first" style={ stepCoords } >
```

Comment:
For now I'm thinking yes, that's the meaning of this component. It's to be used for the initial step of a tour because we want that one to look special, and consistently so.